src/main/java/com/wise23/chariteed/controller/AuthController.java

```diff
@@ -0,0 +1,49 @@
+package com.wise23.chariteed.controller;
+
+import com.wise23.chariteed.model.User;
+import com.wise23.chariteed.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@Controller
+public class AuthController {
+    @Autowired
+    UserService userService;
+
+    @RequestMapping(value = {"/login"}, method = RequestMethod.GET)
+    public String login(){
+        return "auth/login";
+    }
+
+    @RequestMapping(value = {"/register"}, method = RequestMethod.GET)
+    public String register(Model model){
+        model.addAttribute("user", new User());
+        return "auth/register";
+    }
+
+    @RequestMapping(value = {"/register"}, method = RequestMethod.POST)
+    public String registerUser(Model model, @Valid User user, BindingResult bindingResult){
+        if(bindingResult.hasErrors()){
+            model.addAttribute("successMessage", "User registered successfully!");
+            model.addAttribute("bindingResult", bindingResult);
```

Comment:
As far as I understand, no need to re-add bindingResult, unless some kind of redirect is used.

Reply:
I removed the the adding of `bindingResult`